src/lsptoolshost/dotnetRuntime/dotnetRuntimeExtensionApi.ts

```diff
@@ -24,7 +24,7 @@ export interface IDotnetFindPathContext {
 /**
  * https://github.com/dotnet/vscode-dotnet-runtime/blob/main/vscode-dotnet-runtime-library/src/IDotnetAcquireContext.ts
  */
-interface IDotnetAcquireContext {
+export interface IDotnetAcquireContext {
```

Comment:
Is there no npm package for this?

Reply:
I don't believe so - @nagilson there is no NPM package for the .net install tool types right?

Reply:
Unfortunately, no, and this makes me very sad
(we should probably do that)